scout/server/blueprints/variants/controllers.py

```diff
@@ -205,12 +222,84 @@ def str_variants(
     return return_view_data
 
 
-def get_manual_assessments(variant_obj):
+def _populate_assessment_type(assessment_obj, variant_obj, assessment_type, assessment_terms=None):
```

Comment:
I feel like this function is very complicated and I'd like to simplify. I'm going to rewrite it!